### PR TITLE
fix: fetching off-chain claims from did document

### PR DESCRIPTION
**Pull request by Harasz**

### CODE REVIEW

File: lib/LoginStrategy.types.ts [hunk 1]
```diff
@@ -1,18 +1,19 @@
...
   namespace: string;
   owner: string;
-  definition: IRoleDefinition;
+  definition: IRoleDefinitionV2;
 }
 
 export interface OffchainClaim {
   claimType: string;
   claimTypeVersion: number;
-  issuedToken: string;
```
Comment (JGiter) [inline lib/LoginStrategy.types.ts:14]: Why is this?

Comment (Harasz) [reply lib/LoginStrategy.types.ts:14]: There is now `issuedToken` in claim. All data is already decoded from ipfs and token, and we never add this token to decoded data

File: lib/ClaimVerifier.ts [hunk 2]
```diff
@@ -1,8 +1,6 @@
-import { OffchainClaim, DecodedToken } from './LoginStrategy.types';
-import * as jwt from 'jsonwebtoken';
+import { OffchainClaim } from './LoginStrategy.types';
 import { IRoleDefinition } from '@energyweb/credential-governance';
 import { IDIDDocument } from '@ew-did-registry/did-resolver-interface';
-import { ProofVerifier } from '@ew-did-registry/claims';
 
 export class ClaimVerifier {
```
Comment (JGiter) [inline lib/ClaimVerifier.ts:5]: Even if this is not relates to this PR I wonder can ClaimVerifier be replaced by CredentialResolver and ClaimIssuerVerification from vc-verification now?

**Review by JGiter — APPROVED** [review]
Probably it is has to be done in separate task

File: lib/ClaimVerifier.ts (hunk 2 above)
Comment (Harasz) [reply lib/ClaimVerifier.ts:5]: I think so, I will create a ticket

**Pull request merged**